packages/lib-classifier/src/components/Classifier/components/SubjectViewer/components/VariableStarViewer/VariableStarViewer.js

```diff
@@ -68,30 +79,54 @@ const VariableStarViewer = React.forwardRef(function VariableStarViewer(props, r
         visibleSeries={visibleSeries}
       />
       <Box
+        border={zoomEnabled.phasedJSON && { color: 'brand', size: 'xsmall' }}
         gridArea='phasedJSON'
+        style={{ position: 'relative' }}
       >
+        <ZoomEnableButton onClick={() => setAllowPanZoom('phasedJSON')} zooming={zoomEnabled.phasedJSON} />
         <ScatterPlotViewer
           data={phasedJSON.data}
           invertAxes={{ x: false, y: invertYAxis }}
+          margin={{
+            bottom: 50,
+            left: 60,
+            right: 10,
+            top: 30
+          }}
+          setOnPan={setOnPan}
+          setOnZoom={setOnZoom}
           underlays={underlays}
           xAxisLabel={counterpart('VariableStarViewer.phase')}
           xAxisNumTicks={8}
           yAxisLabel={phasedJSON.chartOptions.yAxisLabel}
           yAxisNumTicks={8}
           visibleSeries={visibleSeries}
+          zooming={zoomEnabled.phasedJSON}
         />
       </Box>
       <Box
+        border={zoomEnabled.rawJSON && { color: 'brand', size: 'xsmall' }}
         gridArea='rawJSON'
+        style={{ position: 'relative' }}
```

Comment:
I think `onKeyDown` handling should happen up here, so that you catch events emitted when the Zoom button has focus.